flash/audio/classification/input.py

```diff
@@ -135,3 +147,26 @@ def load_data(
         if root is None:
             root = os.path.dirname(csv_file)
         return super().load_data(data_frame, input_key, target_keys, root, resolver)
+
+
+class AudioClassificationFileInputToSpectrogram(AudioClassificationInput):
```

Comment:
Rather than creating a new input, we could just add the support directly to the existing `spectrogram_loader` and just check if the file has an audio extension